ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py

```diff
@@ -397,3 +397,53 @@ def test_numpy_cbrt(
         subok=True,
         test_values=False,
     )
+
+
+# trunc
```

Comment:
Please add the @handle_cmd_line_args, this allows us to handle with_out, as_variable, etc, by us.